src/DynamoCoreWpf/Utilities/CrashUtilities.cs

```diff
@@ -13,7 +12,7 @@ internal static string GithubNewIssueUrlFromCrashContent(object crashContent)
         {
             var baseUri = new UriBuilder(Configurations.GitHubBugReportingLink);
 
-            var issueTitle = "Crash report from Dynamo {0}";
```

Comment:
we still want to keep the title "Crash report from ..." when it is an actual crash. Can we use "Issue report from ..." only when issue is submitted manually from **Help > Report A Bug**